pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -334,7 +334,7 @@ stdenv.mkDerivation (finalAttrs: {
   src = fetchgit {
     url = "https://git.ffmpeg.org/ffmpeg.git";
     rev = "n${finalAttrs.version}";
-    inherit sha256;
+    inherit hash;
```

Comment:
I think the issue with hashes is here. The problem is the same as I [originally suggested](https://github.com/NixOS/nixpkgs/pull/270429#pullrequestreview-1751434986), but it applies to `hash` rather than the rev (which correctly uses `finalAttrs`)..

You should be able to fix either by overriding `src` together with version, or using the `import ../../ffmpeg/generic.nix { ... }` pattern.

Reply:
Like this?